components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -3417,21 +3417,7 @@
         public static final String GET_MOST_RECENT_REVISION_UUID = "SELECT REVISION_UUID FROM AM_REVISION WHERE " +
                 "API_UUID = ? ORDER BY ID DESC LIMIT 1";
         public static final String GET_REVISION_APIID_BY_REVISION_UUID = "SELECT API_UUID, ID FROM AM_REVISION WHERE REVISION_UUID = ?";
-        static final String GET_API_REVISION_DEPLOYMENTS
-                = "(SELECT t3.NAME, t3.VHOST, t3.REVISION_UUID, MAX(t3.DEPLOYED_TIME) AS DEPLOYED_TIME, " +
-                "SUM(t3.DISPLAY_ON_DEVPORTAL) AS DISPLAY_ON_DEVPORTAL, MAX(t3.DEPLOY_TIME) AS DEPLOY_TIME " +
-                "FROM " +
-                "(SELECT NAME, VHOST, REVISION_UUID, DEPLOYED_TIME, 0 AS DISPLAY_ON_DEVPORTAL, NULL AS DEPLOY_TIME " +
-                "FROM " +
-                "AM_DEPLOYED_REVISION DR " +
-                "UNION " +
-                "SELECT NAME, VHOST, REVISION_UUID, NULL AS DEPLOYED_TIME, DISPLAY_ON_DEVPORTAL, DEPLOYED_TIME as DEPLOY_TIME " +
-                "FROM AM_DEPLOYMENT_REVISION_MAPPING DRM) AS t3 " +
-                "GROUP BY t3.NAME, t3.REVISION_UUID, t3.VHOST)";
-        public static final String GET_REVISIONS_BY_API_UUID = "SELECT ID, AR.REVISION_UUID, DESCRIPTION, CREATED_TIME, " +
-                "CREATED_BY, NAME, VHOST, DISPLAY_ON_DEVPORTAL, DEPLOY_TIME, DEPLOYED_TIME " +
-                "FROM AM_REVISION AR LEFT JOIN " + GET_API_REVISION_DEPLOYMENTS + "AD " +
-                "ON AR.REVISION_UUID = AD.REVISION_UUID WHERE AR.API_UUID = ?";
+        public static final String GET_REVISIONS_BY_API_UUID = "SELECT * FROM AM_REVISION WHERE API_UUID = ?";
```

Comment:
Shall we specify the exact column names to retrieve..

Reply:
Fixed. https://github.com/wso2/carbon-apimgt/pull/10743/commits/f5eef89c31276479036bcdfa88e8ce6e1e692367